### PR TITLE
tweaks to the config file

### PR DESCRIPTION
**Pull request by teolemon**
tweaks to the config file

### CODE REVIEW

File: scripts/sample_export_config.json [hunk 1]
```diff
@@ -113,6 +113,26 @@
...
+      "nutrient_id": 1089,
+      "scale": 0.001
+    },
+    {
+      "name": "vitamin-a_IU_100g",
+      "nutrient_id": 1089,
+      "scale": 1
+    },
+    {
+      "name": "vitamin-C_IU_100g",
```
Comment (KesterTong) [inline scripts/sample_export_config.json:133]: I believe you want lowercase here (`vitamin-c`) based on `https://static.openfoodfacts.org/data/data-fields.txt`.  Or if you have some other convention for imports, `vitamin-a` above is lowercase.

File: scripts/sample_export_config.json [hunk 2]
```diff
@@ -113,6 +113,26 @@
...
+      "scale": 0.001
+    },
+    {
+      "name": "iron_100g",
+      "nutrient_id": 1089,
+      "scale": 0.001
+    },
+    {
+      "name": "vitamin-a_IU_100g",
+      "nutrient_id": 1089,
```
Comment (KesterTong) [inline scripts/sample_export_config.json:129]: This needs updating

File: scripts/sample_export_config.json [hunk 3]
```diff
@@ -113,6 +113,26 @@
...
+      "scale": 1
+    },
+    {
+      "name": "vitamin-C_IU_100g",
+      "nutrient_id": 1089,
```
Comment (KesterTong) [inline scripts/sample_export_config.json:134]: Same here.  In this case there is actually no Vitamin C in IU units, only in mg.

Sometimes the same vitamin appears multiple times in the nutrients definition file (`nutrient.csv`).  In these cases it's helpful to run the `summarize` command and look at the file that is also called `nutrient.csv` in the output of `summarize`.  This gives the nutrients that are actually used in the Branded Food dataset with their frequencies.  This is useful for determining which nutrients to prioritize.

File: scripts/sample_export_config.json (hunk 1 above)
Comment (teolemon) [inline scripts/sample_export_config.json:133]:
```suggestion
      "name": "vitamin-c_iu_100g",
```

File: scripts/sample_export_config.json [hunk 4]
```diff
@@ -113,6 +113,26 @@
...
+      "nutrient_id": 1087,
+      "scale": 0.001
+    },
+    {
+      "name": "iron_100g",
+      "nutrient_id": 1089,
+      "scale": 0.001
+    },
+    {
+      "name": "vitamin-a_IU_100g",
```
Comment (teolemon) [inline scripts/sample_export_config.json:128]:
```suggestion
      "name": "vitamin-a_iu_100g",
```

File: scripts/sample_export_config.json [hunk 5]
```diff
@@ -113,6 +113,26 @@
...
+      "nutrient_id": 1087,
+      "scale": 0.001
+    },
+    {
+      "name": "iron_100g",
+      "nutrient_id": 1089,
+      "scale": 0.001
+    },
+    {
+      "name": "vitamin-a_iu_100g",
```
Comment (teolemon) [inline scripts/sample_export_config.json:128]:
```suggestion
      "name": "vitamin_a_iu_100g",
```

File: scripts/sample_export_config.json [hunk 6]
```diff
@@ -113,6 +113,26 @@
...
+      "nutrient_id": 1089,
+      "scale": 1
+    },
+    {
+      "name": "vitamin-c_iu_100g",
+      "nutrient_id": 1089,
```
Comment (teolemon) [inline scripts/sample_export_config.json:134]:
```suggestion
      "name": "vitamin-c_mg_100g",
      "nutrient_id": 1162,
```

File: scripts/sample_export_config.json [hunk 7]
```diff
@@ -113,6 +113,26 @@
...
+      "scale": 0.001
+    },
+    {
+      "name": "iron_100g",
+      "nutrient_id": 1089,
+      "scale": 0.001
+    },
+    {
+      "name": "vitamin-a_iu_100g",
+      "nutrient_id": 1089,
```
Comment (teolemon) [inline scripts/sample_export_config.json:129]:
```suggestion
      "nutrient_id": 1104,
```

**Comment by KesterTong** [comment]
Looks like this is ready to merge, except the comments need removing (JSON doesn't support comments)

**Comment by teolemon** [comment]
Added them on purpose to force discussion and prevent merging, sorry. Will commit a clean version soon.

**Comment by KesterTong** [comment]
You still want to merge this?  I assume the import is done but it might be useful to have this checked in for reference

**Comment by teolemon** [comment]
I'm closing this one in favor of the newer one

**Pull request closed without merging**